app/server/api/organisation_api.py

```diff
@@ -172,6 +180,9 @@ def post(self, organisation_id):
             else:
                 return make_response(jsonify(cic_response_object)), cic_response_code
 
+        if AccessControl.has_exact_role(g.user.roles, 'ADMIN', 'superadmin'):
+            g.user.add_user_to_organisation(new_organisation, is_admin=True)
+
```

Comment:
if a superadmin creates an organisation via GUI, we want to add the user to the organisation